build/config/SignToolData.json

```diff
@@ -127,6 +127,7 @@
         "System.IO.FileSystem.DriveInfo.dll",
         "System.IO.FileSystem.Primitives.dll",
         "System.IO.Pipes.dll",
+        "System.Xml.ReaderWriter.dll",
```

Comment:
OK, but why wasn't this facade being linked in before? Did upgrading the package pull in a facade that didn't exist for 4.6/netcore 1.0?